src/pages/GameInstancePage.tsx

```diff
@@ -0,0 +1,115 @@
+// TODO: this page will be removed with the addition of user-based logic.
```

Comment:
Oh sounds like this addresses my above comments! Happy to discuss it anyway or feel free to just resolve those (would be nice if we could get a ticket to track that though)

Reply:
done https://gaming-backlog.atlassian.net/browse/GB-75 can be in the cleanup milestone